contrib/travis_fastfail.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+curlhdr="Accept: application/vnd.travis-ci.2+json"
+endpoint="https://api.travis-ci.org/repos/$TRAVIS_REPO_SLUG"
+
+# Fail fast for superseded builds to PR's
+if ! [ "$TRAVIS_PULL_REQUEST" = "false" ]; then
+  if ! [ \"$TRAVIS_BUILD_NUMBER\" = $(curl -H "$curlhdr" $endpoint/builds?event_type=pull_request | \
+      jq ".builds | map(select(.pull_request_number == $TRAVIS_PULL_REQUEST))[0].number") ]; then
+    echo "There are newer queued builds for this pull request, failing early."
+    exit 1
+  fi
+fi
+
+# And for non-latest push builds in branches other than master or release*
+case $TRAVIS_BRANCH in
```

Comment:
What is `TRAVIS_BRANCH` set to for PR? Especially the ones that come from a different repo.?